src/content/docs/en/guides/troubleshooting.mdx

```diff
@@ -121,7 +121,7 @@ Then check your import statement:
 
 - Does your import have the same name as the imported component? (Check your component name and that it [follows the `.astro` syntax](/en/core-concepts/astro-syntax/#differences-between-astro-and-jsx).)
 
-- Have you included the extension in the import? (Check that your imported file contains an extension. e.g. `.astro`, `.md`, `.vue`, `.svelte`. Note: File extensions are **not** required for `.js(x)` and `.ts(x)` files only.)
+- Have you included the extension in the import? (Check that your imported file contains an extension. e.g. `.astro`, `.md`, `.vue`, `.svelte`. Note: File extensions are **optional** for .js(x) and .ts(x) files.)
```

Comment:
`optional` means a new Developer has an uncertain choice for extensions in Astro config. Its best to give them a straight advice what/**what not** to do so that they don't have to question themselves at first place

Reply:
> `optional` means a new Developer has an uncertain choice for extensions in Astro config. Its best to give them a straight advice what/**what not** to do so that they don't have to question themselves at first place

In all fairness the change is subjective and can receive positive feedback as well

Reply:
Thank you for these details! It's true that I hadn't necessarily thought of this interpretation.